docs/projects/version-control/unsupported-config-as-code-scenarios.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Unsupported Configuration as Code Scenarios
+description: Unsupported scenarios for the Configuration as Code feature in Octopus Deploy.
+position: 50 
+---
+
+The Configuration as Code feature is designed to give you the benefits of source control, branching, reverting, and pull requests while being able to use your tool of choice to manage your processes (and eventually) variables. While it has many benefits, there are some unsuitable use cases and scenarios. This document will describe each one as well as provide alternatives.
+
+## Core Design Decision
+
+The core design decision is that each project in each space has a unique folder in a git repository. A git repository can store several projects across several spaces or store a single project in a single space. Each project must have a unique folder in a git repository because of all the scaffolding data referenced by a project.  
+
+That scaffolding data includes (but is not limited to):
+
+- Environments
+- Tenant Tags
+- Worker Pools
+- Feeds
+- Step Templates
+- Channels / Lifecycles
+
+That data is not stored in source control because it is shared across multiple projects.
+
+An error will occur when Octopus Deploy attempts to load a process from source control with one or more of those items missing. You'll be unable to create releases until those errors are resolved.
+
+## Syncing Multiple Instances
+
+The configuration as code feature is not designed to allow two or more projects on different instances to point to the same folder. We've seen our users attempt to use Configuration as Code to keep the deployment processes in sync across multiple instances. That scenario is unsupported.
+
+While it may work initially, it will be harder and harder to manage over time. You will need to keep _all_ the scaffolding data in-sync across multiple instances. That is [easier said than done](/docs/administration/sync-instances/index.md).  Step templates will be the most difficult, as having the same step template on all instances, the version has to match. Otherwise, you'll have to worry about settings such as parameters, scripts, package versions, feeds, and more.
```

Comment:
```suggestion
While it may work initially, it will be harder and harder to manage over time. You will need to keep _all_ the scaffolding data in sync across multiple instances. That is [easier said than done](/docs/administration/sync-instances/index.md).  Step templates will be the most difficult, as having the same step template on all instances, the version has to match. Otherwise, you'll have to worry about settings such as parameters, scripts, package versions, feeds, and more.
```